### ISSUE TITLE
Changing the data source in the live demo

### ISSUE THREAD

**Issue by pchampin** [vitalis-wiens/donatello-pipelines#1]
When I try to change the SPARQL query in the Wikidata or DBPedia example, it does not work. More precisely, when I close the configuration dialog, and execute it again, I get the results from the original configuration. When I open the configuration dialog again, my changes have been reverted to the original.

**Comment by vitalis-wiens** [comment]
Hi, thanks.
I missed to update the query for building the pipeline.
I have added this option to test the configuration and see if the "test query function" is able to retrieve the data.

However, you could configure the pipeline, and export it as a zip file, and then update the query in the corresponding data access component (e.g., Implementation/DataAccessComponent/DBPediaLoader.js)

**Comment by pchampin** [comment]
> However, you could configure the pipeline, and export it as a zip file

Yes, I ended up doing that, actually :) But doing this in the GUI would be more convenient, eventually :)
